packages/logger/package.json

```diff
@@ -27,15 +27,15 @@
     "dependencies": {
         "chalk": "4.1.2",
         "lodash": "4.17.21",
-        "logform": "2.3.2",
         "winston": "3.3.3",
         "winston-transport": "4.4.1"
     },
     "devDependencies": {
         "@types/debug": "4.1.7",
         "@types/lodash": "4.14.176",
         "@types/vscode": "1.63.1",
-        "jest-extended": "1.2.0"
+        "jest-extended": "1.2.0",
+        "logform": "2.3.2"
```

Comment:
Adding a note that this is only required because the type `Format` is not exposed by `winston` - it's used here: 
https://github.com/SAP/open-ux-tools/blob/34beb9f8899fc997a12351dfd0c34d78c31d8776/packages/logger/src/winston-logger/adapter.ts#L8
https://github.com/SAP/open-ux-tools/blob/34beb9f8899fc997a12351dfd0c34d78c31d8776/packages/logger/src/winston-logger/adapter.ts#L46